### ISSUE TITLE
Which is the test split of FEAFA dataset?

### ISSUE THREAD

**Issue by gerwang** [deepmo24/CPEM#10]
Hi, 
In the released code, it seems that all the FEAFA dataset is used for training. I wonder which test split was used to generate Tab. 1 in the paper? Thank you!

**Comment by deepmo24** [comment]
Hi, we divide 123 FEAFA videos into two parts,  the last 103 videos as the training set and the first 20 videos as the test set, which is described in the [supplementary file](https://github.com/deepmo24/CPEM/blob/main/assets/supplementary.pdf).

**Comment by gerwang** [comment]
Thank you! I will try to obtain the peak expression frames using threshold. I wonder if it is possible to make the data_files (npy) of feafa public?
